lib/trellis/vagrant.rb

```diff
@@ -72,14 +72,29 @@ def mount_options(mount_type, dmode:, fmode:)
   end
 end
 
-def post_up_message
+def post_up_message(trellis_config)
   msg = 'Your Trellis Vagrant box is ready to use!'
   msg << "\n* Composer and WP-CLI commands need to be run on the virtual machine"
   msg << "\n  for any post-provision modifications."
   msg << "\n* You can SSH into the machine with `vagrant ssh`."
   msg << "\n* Then navigate to your WordPress sites at `/srv/www`"
   msg << "\n  or to your Trellis files at `#{ANSIBLE_PATH_ON_VM}`."
 
+  if trellis_config.wordpress_sites.any?
+    msg << "\n\n\e[35m🚀 Development URLs: \e[0m"
+    trellis_config.wordpress_sites.each do |site_name, site|
+      site_url = site['site_hosts'].first
+      site_url = site_url.is_a?(Hash) ? site_url['canonical'] : site_url
+      site_url = site['ssl']['enabled'] ? "https://#{site_url}" : "http://#{site_url}"
```

Comment:
Not sure if its possible, but to be very defensive this could be:

```ruby
site.dig('ssl', 'enabled') ? "https://#{site_url}" : "http://#{site_url}"
```

That way it will gracefully handle either ssl or enabled not existing

Reply:
Fixed, and I changed emojis for more consistent spacing in different terminals.